web/packages/shared/components/Controls/MultiselectMenu.story.tsx

```diff
@@ -0,0 +1,141 @@
+  disabled?: boolean;
+  disabledTooltip?: string;
+}[] = [
+  { value: 'option-1', label: 'Option 1' },
+  { value: 'option-2', label: 'Option 2' },
+  { value: 'option-3', label: 'Option 3' },
+  { value: 'option-4', label: 'Option 4' },
+];
+
+const optionsWithCustomLabels: typeof options = [
+  {
+    value: 'option-1',
+    label: <strong>Bold Option 1</strong>,
+  },
+  {
+    value: 'option-3',
+    label: <em>Italic Option 3</em>,
+  },
+];
+
+export default {
+  title: 'Shared/Controls/MultiselectMenu',
+  component: MultiselectMenu,
+  argTypes: {
+    buffered: {
+      control: { type: 'boolean' },
+      description: 'Buffer selections until "Apply" is clicked',
+      table: { defaultValue: { summary: 'false' } },
+    },
+    showIndicator: {
+      control: { type: 'boolean' },
+      description: 'Show indicator when there are selected options',
+      table: { defaultValue: { summary: 'true' } },
+    },
+    showSelectControls: {
+      control: { type: 'boolean' },
+      description: 'Show select controls (Select All/Select None)',
+      table: { defaultValue: { summary: 'true' } },
+    },
+    label: {
+      control: { type: 'text' },
+      description: 'Label for the multiselect',
+    },
+    tooltip: {
+      control: { type: 'text' },
+      description: 'Tooltip for the label',
+    },
+    selected: {
+      control: false,
+      description: 'Currently selected options',
+      table: { type: { summary: 'T[]' } },
+    },
+    onChange: {
+      control: false,
+      description: 'Callback when selection changes',
+      table: { type: { summary: 'selected: T[]' } },
+    },
+    options: {
+      control: false,
+      description: 'Options to select from',
+      table: {
+        type: {
+          summary:
+            'Array<{ value: T; label: string | ReactNode; disabled?: boolean; disabledTooltip?: string; }>',
+        },
+      },
+    },
+  },
+  args: {
+    label: 'Select Options',
+    tooltip: 'Choose multiple options',
+    buffered: false,
+    showIndicator: true,
+    showSelectControls: true,
+  },
+  parameters: { controls: { expanded: true, exclude: ['userContext'] } },
+  render: (args => {
+    const [selected, setSelected] = useState<string[]>([]);
+    return (
+      <Flex alignItems="center" minHeight="100px">
+        <MultiselectMenu {...args} selected={selected} onChange={setSelected} />
+      </Flex>
+    );
+  }) satisfies StoryFn<typeof MultiselectMenu<OptionValue>>,
+} satisfies Meta<typeof MultiselectMenu<OptionValue>>;
+
+type Story = StoryObj<typeof MultiselectMenu<OptionValue>>;
+
+const Default = {
+  args: { options },
+} satisfies Story;
+
+const WithCustomLabels = {
+  args: { options: optionsWithCustomLabels },
+} satisfies Story;
+
+const WithDisabledOption = {
+  args: {
+    options: [
+      ...options,
+      {
+        value: 'option-5',
+        label: 'Option 5',
+        disabled: true,
+        disabledTooltip: 'Option is disabled',
```

Comment:
I'd adjust the message to say something like "Option is disabled because of foo" or lorem ipsum to show that it's just an example text an not something that is shown in every `MultiselectMenu`.

Reply:
gotcha, makes sense!